config.json

```diff
@@ -1,24 +1,69 @@
 {
     "port": 8000,
     "listenOn": [],
-    "regions": {
-        "ap-northeast-1": ["s3.ap-northeast-1.amazonaws.com"],
-        "ap-southeast-1": ["s3.ap-southeast-1.amazonaws.com"],
-        "ap-southeast-2": ["s3.ap-southeast-2.amazonaws.com"],
-        "eu-central-1": ["s3.eu-central-1.amazonaws.com",
-                         "s3.eu.central-1.amazonaws.com"],
-        "eu-west-1": ["s3.eu-west-1.amazonaws.com"],
-        "sa-east-1": ["s3.sa-east-1.amazonaws.com"],
-        "us-east-1": ["s3.amazonaws.com",
-                      "s3-external-1.amazonaws.com",
-                      "s3.us-east-1.amazonaws.com"],
-        "us-west-1": ["s3.us-west-1.amazonaws.com"],
-        "us-west-2": ["s3-us-west-2.amazonaws.com"],
-        "us-gov-west-1": ["s3-us-gov-west-1.amazonaws.com",
-                          "s3-fips-us-gov-west-1.amazonaws.com"],
-        "localregion": ["localhost"],
-        "test-region": ["s3.scality.test"],
-        "docker-region": ["s3.docker.test"]
+    "locationConstraints": {
+        "aws-us-east-1": {
+            "type": "aws_s3",
+            "legacyAwsBehavior": true,
+            "details": {
+                "region": "us-east-1",
```

Comment:
@dora-korpar @LaurenSpiegel - I guess this `region` is for the location of the bucket "premadebucket" in the real AWS.
So it will be possible to have` "legacyAwsBehavior": true,` with `"region": "us-east-1",`
Right?
_Suggestion:_ Won't it be clearer to have `legacyAwsBehavior` in the `details` block with all the bucket's informations.